test/testPlanarSSSP.cpp

```diff
@@ -0,0 +1,106 @@
+#include <gtest/gtest.h>
```

Comment:
Please also:
1. provide `benchmark/benchmarkShortestPath.cpp` prepared on the basis of `benchmarkMinimumSpanningTree.cpp` (especially g6 reading) + using Dijkstra vs. your algorithms (as MST has Boruvka, Prim etc.)
2. run them on all (!) planar graphs datasets from https://users.cecs.anu.edu.au/~bdm/data/graphs.html - they should return exactly the same results for all methods